cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,42 @@
-describe('', () => {
-  before(() => {
+const faker = require('faker');
+let article; 
+let user;
 
+describe('Article flow suit', () => {
+  beforeEach(() => {
+    cy.task('generateArticle').then(generateArticle => {
+      article = generateArticle;
+    });
+    user = {
+      username: 'joebin1'
+    };
+    cy.login();
   });
 
-  it('', () => {
-
+  it('should create an article', () => {
+    cy.createArticle(article.title, article.description, article.body);
```

Comment:
It would be nice to create an article "manually" without an additional method first. In the second check you can use the custom command